utils/utils.jsx

```diff
@@ -1333,50 +1333,63 @@ export function canCreateCustomEmoji(user) {
     return true;
 }
 
-export function isValidPassword(password) {
+export function getPasswordConfig() {
+    return {
+        isEnterprise: global.window.mm_config.BuildEnterpriseReady === 'true',
+        isLicensed: global.window.mm_license.IsLicensed === 'true',
+        isPasswordRequirements: global.window.mm_license.PasswordRequirements === 'true',
+        minimumLength: parseInt(global.window.mm_config.PasswordMinimumLength, 10),
+        requireLowercase: global.window.mm_config.PasswordRequireLowercase === 'true',
+        requireUppercase: global.window.mm_config.PasswordRequireUppercase === 'true',
+        requireNumber: global.window.mm_config.PasswordRequireNumber === 'true',
+        requireSymbol: global.window.mm_config.PasswordRequireSymbol === 'true',
+    }
+}
```

Comment:
By hiding the static access in this helper function, my test suites can easily sub in config, and the `isValidPassword(...)` all type coercion can be factored out of `isValidPassword(...)`, which makes it easier to understand.